src/setup/cortx_deploy

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+function log_debug {
+    [[ -z $DEBUG || $DEBUG != 1 ]] && return
+
+    echo "log: $*"
+}
+
+function usage {
+    echo "usage: $(basename $0) -n <node id> [-f <solution conf>]"
+    echo "where:"
+    echo "node id       - Unique Node ID"
+    echo "solution conf - Solution conf dir containing cortx.yaml and cluster.yaml"
+    echo
+    exit 1
+}
+
+#[ $(id -u) -ne 0 ] && echo "error: run this command as root user" && exit 1
+
+# set MOCK to false by default
+MOCK=false
+
+# Constants
+SOLUTION_CONF="/etc/cortx/solution"
```

Comment:
Change this to `SOLUTION_CONFIG`

Reply:
Please define other values as well:
CONFIG_INFO = "yaml://$SOLUTION_CONFIG/config.yaml"
CLUSTER_INFO = "yaml://$SOLUTION_CONFIG/cluster.yaml"